cmd/liqo-controller-manager/main.go

```diff
@@ -495,6 +510,40 @@ func main() {
 		}
 	}
 
+	// Connect to the IPAM server if specified.
+	if *ipamServer != "" {
+		klog.Infof("connecting to the IPAM server %q", *ipamServer)
+		dialctx, cancel := context.WithTimeout(ctx, 10*time.Second)
+		connection, err := grpc.DialContext(dialctx, *ipamServer,
+			grpc.WithTransportCredentials(insecure.NewCredentials()), grpc.WithBlock())
+		cancel()
+		if err != nil {
+			klog.Errorf("failed to establish a connection to the IPAM %q", *ipamServer)
+			os.Exit(1)
+		}
+		ipamClient = ipam.NewIpamClient(connection)
+	}
+
+	networkReconciler := &networkctrl.NetworkReconciler{
+		Client:     mgr.GetClient(),
+		Scheme:     mgr.GetScheme(),
+		IpamClient: ipamClient,
+	}
+	if err = networkReconciler.SetupWithManager(mgr); err != nil {
+		klog.Errorf("Unable to start the networkReconciler", err)
+		os.Exit(1)
+	}
+
+	ipReconciler := &ipctrl.IPReconciler{
+		Client:     mgr.GetClient(),
+		Scheme:     mgr.GetScheme(),
+		IpamClient: ipamClient,
+	}
+	if err = ipReconciler.SetupWithManager(mgr); err != nil {
+		klog.Errorf("Unable to start the ipReconciler", err)
+		os.Exit(1)
+	}
```

Comment:
Should we avoid starting the controllers if the ipam server is not defined? i.e. when the liqo network is disabled